docs/one-click-integrations/github-integration.md

```diff
@@ -135,9 +135,9 @@ The table below lists the data that the GitHub integration collects from your Gi
 </tr>
 </table>
 
-<sup><span id="commit-author-date">1</span></sup>: Pulse uses the commit author's date since it is more accurate. The committer date can be changed (e.g.: rebases) and stop reflecting the real creation date of the change.
+<sup><span id="commit-author-date">1</span></sup>: Pulse uses the commit author's date since it's more accurate. The committer date can be changed (e.g.: rebases) and stop reflecting the real creation date of the change.
```

Comment:
@prcr I didn't fix the first one because I did it in [#171](https://github.com/codacy/pulse-user-docs/pull/171/commits/a9e9b80f309ef4dab33eaf21f1083bc414db3744), as I was working on that file.

Regarding the "rebase", I was getting a Vale error not related to the "e.g.". I was considering adding "rebase" to the accept list. WDYT?

Reply:
I just checked and yes, Vale doesn't recognize the word "rebase" so we must add it to the accept list. :+1: 

Regarding the "e.g.", [the Microsoft Style Guide recommends avoiding it](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg). I think that's probably because many people don't know [what it means](https://en.wiktionary.org/wiki/e.g.) since it's a bit formal.